index.js

```diff
@@ -1,124 +0,0 @@
-'use strict';
-
-var logUtil = require('./logUtil');
-var linesUtil = require('./linesUtil');
-var config = require('./config');
-
-function getLogLevel() {
-	var level = process.env.LOG_LEVEL;
-	if (level) {
-		level = config._logLevels[level.toLowerCase()];
-	} else {
-		//default log level
-		level = 7;
-	}
-	return level;
-}
-
-function meetsLogLevelRequirement(loggerLevel) {
-	return config._logLevels[loggerLevel] <= getLogLevel();
-}
-
-var m = {
-	//////////////////
-	// main methods //
-	//////////////////
-
-	// default
-	d: function() {
-		if (meetsLogLevelRequirement('default')) {
-			logUtil.logWithColor(config.colors.default, arguments);
-		}
-	},
-	// success
-	s: function() {
-		if (meetsLogLevelRequirement('success')) {

```

Comment:
this format was tedious... if I ever want to change anything with the functions it is a lot of copy paste... this leads to lots of work, and potential for creating bugs.